src/Protocol/ActivityPub/Transmitter.php

```diff
@@ -293,27 +319,41 @@ public static function getOutbox(array $owner, int $page = null, string $request
 			$data['orderedItems'] = $list;
 		}
 
+		if (!empty($cachekey)) {
```

Comment:
According to the description of the parameter, I believe you should have checked the value of `$nocache` here.